appleauth/auth_source.go

```diff
@@ -175,3 +175,18 @@ func (*InputAppleIDFastlaneSource) Fetch(conn *devportalservice.AppleDeveloperCo
 		},
 	}, nil
 }
+
+func appSpecificPasswordFavouringConnection(conn *devportalservice.AppleIDConnection, passwordFromInput string) string {
+	appSpecificPassword := ""
+
+	if passwordFromInput != "" {
+		appSpecificPassword = passwordFromInput
+	}
```

Comment:
```suggestion
	appSpecificPassword = passwordFromInput
```
This would be equivalent, as far as I see